classes/Access.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * 2007-2018 PrestaShop
+ * 2007-2018 PrestaShop.
```

Comment:
Why is it adding dots?

Reply:
because of this rule `phpdoc_annotation_without_dot [@Symfony]` I think

Reply:
It's messing with our file headers :\